manager/src/main/java/io/gingersnapproject/Caches.java

```diff
@@ -176,6 +178,17 @@ public Uni<Boolean> remove(String name, String key) {
       return Uni.createFrom().item(Boolean.FALSE);
    }
 
+   public void removeCache(String ruleName) {
```

Comment:
Can't we just do `@Observes RuleEvents.EagerRuleRemoved ev` here and avoid calling a method from `DatabaseHandler`?

Reply:
Yeah maybe yes, let me check that the exec order doesn't matter.

Reply:
seems safe, changed to use events